src/main/views/menu.html

```diff
@@ -1,3 +1,4 @@
-
-<a href="/import" >Import Case Definition</a> 
-<a href="/jurisdiction" >Manage User Profiles</a> 
+<div class="padding">
+  <a href="/import" >Import Case Definition</a> 
```

Comment:
Not sure if the linting will pick this up, but there is an extra space before the first `>`.